docs/themes/thxvscode/assets/scss/includes/_header.scss

```diff
@@ -228,6 +228,9 @@ iframe[name='google_conversion_frame'] {
   position: relative;
   a {
     color: $banner-link-color;
+    &:visited {
+      color: #ac7ee1;
```

Comment:
It would be nice to define this color in the theme, similar to `$banner-link-color` above.

Reply:
Agreed.